docs/architecture/adr-069-flexible-node-intitalization.md

```diff
@@ -0,0 +1,164 @@
+# ADR 069: Flexible Node Initialization
+
+## Changlog
+
+- 2021-06-09: Initial Draft (@tychoish)
+
+## Status 
+
+Proposed. 
+
+## Context
+
+In an effort to support [Go-API-Stability](./adr-060-go-api-stability.Md),
+during the 0.35 development cycle, we have attempted to reduce the total size
+of the API surface area by moving most of the interface of the `node` package
+into unexported functions, as well as moving the reactors to an `internal`
+package. Having this coincide with the 0.35 release made a lot of sense
+because these interfaces were _already_ changing as a result of the `p2p`
+[refactor](./adr-061-p2p-refactor-scope.md), so it made sense to think a bit
+more about how tendermint exposes this API.
+
+It is not immediately intuitive that the nexus of this change is in the
+relationship between the `rpc` package and the `node` package, particularly
```

Comment:
Can you either explain this a bit more or link to a code sample/line number that illustrates this?